spec/fixtures/csv/locations_upload_invalid_headers.csv

```diff
@@ -0,0 +1,3 @@
+Invalid,Address line 2,Address line 3,City,County,Postcode,IP address,IP address,IP address,IP address,IP address
+122,London road,,London,Greater London,LE2 0EN
+101,Avenue  street,,Leicester,Leicestershire,SE2 9TT
```

Comment:
Git complains about there not being a newline.Maybe just add one to make it stop 😁